### DISCUSSION TITLE
New Feature: Markdown Support

### DISCUSSION THREAD

**Discussion by steven-tey** [steven-tey/novel#57] [Announcements]
Just added markdown support to Novel.sh ✨

https://github.com/steven-tey/novel/assets/28986134/59f138c5-692f-4159-8e38-d28117e74e14

You can now copy your content from Novel and it'll automatically be transformed into Markdown that you can paste anywhere.

Shoutout to @aguingand's amazing `tiptap-markdown` library that made this possible 💪

Go give them a star: https://github.com/aguingand/tiptap-markdown

**Comment by NoureddinBakir** [comment]
So excited for this! I just sat down to work on this, thinking it hasn't been solved yet as per the previous discussion. Thank you @steven-tey and @aguingand you're both amazing!

**Comment by eduwass** [comment]
This looks great!

Do you guys support this the other way round, e.g: being able to populate Novel with Markdown content?

**Reply by EryouHao** [reply]
Is this way supported now?

**Reply by anishjoshi1999** [reply]
@steven-tey  have a similar question.Does the Novel package support Markdown and LaTeX features similar to Notion?I’m considering building an editor with both Markdown and LaTeX functionalities using Novel. Are these capabilities supported by the current version of the package?
